app/views/layouts/application.html.erb

```diff
@@ -24,5 +24,7 @@
      client side hydration -->
 <%= redux_store_hydration_data %>
 
+<%= react_component "Footer" %>
+
```

Comment:
Though it doesn't matter in this case, by better to keep `<%= redux_store_hydration_data %>` at the end.